ext/session/mod_files.sh

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/env bash
 
 if [[ "$2" = "" ]] || [[ "$3" = "" ]]; then
-       echo "Usage: $0 BASE_DIRECTORY DEPTH HASH_BITS"
+       echo "Usage: $0 BASE_DIRECTORY DEPTH BITS_PER_CHAR"
        echo "BASE_DIRECTORY will be created if it doesn't exist"
        echo "DEPTH must be an integer number >0"
-       echo "HASH_BITS(session.hash_bits_per_character) should be one of 4, 5, or 6"
+       echo "For PHP < 7.1.0. BITS_PER_CHAR(session.hash_bits_per_character) should be one of 4, 5, or 6."
+       echo "For PHP >= 7.1.0. BITS_PER_CHAR(session.sid_bits_per_character) should be one of 4, 5, or 6."
+       # http://php.net/manual/en/session.configuration.php#ini.session.hash-bits-per-character
+       # http://php.net/manual/en/session.configuration.php#ini.session.sid-bits-per-character
```

Comment:
As this is targeting the master branch, there is no need to retain mention of PHP < 7.1 here.

Reply:
Ok, you know how it fits better.